### DISCUSSION TITLE
The tool does not work on site anymore

### DISCUSSION THREAD

**Discussion by Eriaku69** [aclap-dev/video-downloadhelper#1243] [Help and Q&A]
### Your question

Hello, 

I was using the tool on a website and it works perfectly, since yesterday evening. Just a second after one download, the next one starting to fail. I have the the app and the license of videodownloadhelper. Could you please help me?

### Addon & CoApp version and details

version: 9.1.0.44
target: mozilla
channel: stable
lang: it
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"accepted":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:130.0) Gecko/20100101 Firefox/130.0
{
    "valid_license_message_has_been_displayed": true,
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "open_count_store": 8,
    "successfull_dl": 1
}

**Comment by Mattmjs** [comment]
Help can be offered if you share a link to the video. If it's a private video, click the 3 dot button next to download and look down
to details. Click this and a page will open with more information. Copy this as text ( not an image ) and paste it here.

**Reply by Eriaku69** [reply]
Hi, thanks for the tip!

here the details

<html><body>

Tab | 1 (incognito: false)
-- | --
Aletta Ocean's Official Website
https://alettaoceanlive.com/members/scenes/215-4K50p-webvideo_vids.html with 10 headers
/content/images/no-thumbnail.png
false
https://api.xvid.com/v1/playlists/hls.m3u8?file_id=627ecb38e4b096a69ddb81ae,627ecb38e4b096a69ddb81b1,627ecb38e4b096a69ddb81b2,627ecb38e4b096a69ddb81b3&client_id=61fa64e6e4b0aba28037baef&autograph_tag=jspeed321___at___gmail___dot___com&ip=1609304554&anti_fraud=true&expiry_time=1726185600&multi_use=true&redirect=true&signature=96f21c6f3fd23f0601eb15d7b6f8d56876c59dc7695a1ed2567f18841b899bf4
https://autograph.xvid.com/api/v1/segments/job_id/627ecb38e4b096a69ddb81ab/base/61efb091e4b0a2be5fde0dd0%3Barchive%3B/tag/jspeed321___at___gmail___dot___com/crypt_tag/kf9ptg1jyB9ZtN_Z5Gy7DAje-NItsrQ_G0paqAtRVyJid7v3jY3aYui9_XhXB10aFakd6LUj9hmQtSTiXVtbCT/client_id/61fa64e6e4b0aba28037baef/user_id/61efb091e4b0a2be5fde0dd0/expiry_time/1726272000/rip/1609304554/file_id/627ecb38e4b096a69ddb81b3/host/cdn-eu-west-1.xvid.com/sign/a23f2d9969a8c2291fa06b676da198def49b4e40/map/true/url/627ecb38e4b096a69ddb81ab/output_3/oKfPDIMP82WCVdmAKNuM8xryDo3m6V6bRu_video.m3u8
https://autograph.xvid.com/api/v1/segments/job_id/627ecb38e4b096a69ddb81ab/base/61efb091e4b0a2be5fde0dd0%3Barchive%3B/tag/jspeed321___at___gmail___dot___com/crypt_tag/kf9ptg1jyB9ZtN_Z5Gy7DAje-NItsrQ_G0paqAtRVyJid7v3jY3aYui9_XhXB10aFakd6LUj9hmQtSTiXVtbCT/client_id/61fa64e6e4b0aba28037baef/user_id/61efb091e4b0a2be5fde0dd0/expiry_time/1726272000/rip/1609304554/file_id/627ecb38e4b096a69ddb81ae/host/cdn-eu-west-1.xvid.com/sign/e8aaa81716a8389d1bcbb223553860794d1625c8/map/true/url/627ecb38e4b096a69ddb81ab/output_0/_qIeHereo-cL4aVRMEE7mbB9tLX0SfModH_audio.m3u8
Mp4 b:Hls p:hls 1484.8400000000013s
AAC at unknown
H264 at 12251000, fps: ?, q:2160, {"width":3840,"height":2160}
https://api.xvid.com/v1/playlists/hls.m3u8?file_id=627ecb38e4b096a69ddb81ae,627ecb38e4b096a69ddb81b1,627ecb38e4b096a69ddb81b2,627ecb38e4b096a69ddb81b3&client_id=61fa64e6e4b0aba28037baef&autograph_tag=jspeed321___at___gmail___dot___com&ip=1609304554&anti_fraud=true&expiry_time=1726185600&multi_use=true&redirect=true&signature=96f21c6f3fd23f0601eb15d7b6f8d56876c59dc7695a1ed2567f18841b899bf4
https://autograph.xvid.com/api/v1/segments/job_id/627ecb38e4b096a69ddb81ab/base/61efb091e4b0a2be5fde0dd0%3Barchive%3B/tag/jspeed321___at___gmail___dot___com/crypt_tag/kf9ptg1jyB9ZtN_Z5Gy7DAje-NItsrQ_G0paqAtRVyJid7v3jY3aYui9_XhXB10aFakd6LUj9hmQtSTiXVtbCT/client_id/61fa64e6e4b0aba28037baef/user_id/61efb091e4b0a2be5fde0dd0/expiry_time/1726272000/rip/1609304554/file_id/627ecb38e4b096a69ddb81b1/host/cdn-eu-west-1.xvid.com/sign/66c36ffa8e400ff0b4f9aa6d29a721c24fe02d35/map/true/url/627ecb38e4b096a69ddb81ab/output_1/lAPgidL-taMQtgLE4Hb4aPdQKWcBI9Bf1R_video.m3u8
https://autograph.xvid.com/api/v1/segments/job_id/627ecb38e4b096a69ddb81ab/base/61efb091e4b0a2be5fde0dd0%3Barchive%3B/tag/jspeed321___at___gmail___dot___com/crypt_tag/kf9ptg1jyB9ZtN_Z5Gy7DAje-NItsrQ_G0paqAtRVyJid7v3jY3aYui9_XhXB10aFakd6LUj9hmQtSTiXVtbCT/client_id/61fa64e6e4b0aba28037baef/user_id/61efb091e4b0a2be5fde0dd0/expiry_time/1726272000/rip/1609304554/file_id/627ecb38e4b096a69ddb81ae/host/cdn-eu-west-1.xvid.com/sign/e8aaa81716a8389d1bcbb223553860794d1625c8/map/true/url/627ecb38e4b096a69ddb81ab/output_0/_qIeHereo-cL4aVRMEE7mbB9tLX0SfModH_audio.m3u8
Mp4 b:Hls p:hls 1484.8400000000013s
AAC at unknown
H264 at 9090000, fps: ?, q:1080, {"width":1920,"height":1080}
https://api.xvid.com/v1/playlists/hls.m3u8?file_id=627ecb38e4b096a69ddb81ae,627ecb38e4b096a69ddb81b1,627ecb38e4b096a69ddb81b2,627ecb38e4b096a69ddb81b3&client_id=61fa64e6e4b0aba28037baef&autograph_tag=jspeed321___at___gmail___dot___com&ip=1609304554&anti_fraud=true&expiry_time=1726185600&multi_use=true&redirect=true&signature=96f21c6f3fd23f0601eb15d7b6f8d56876c59dc7695a1ed2567f18841b899bf4
https://autograph.xvid.com/api/v1/segments/job_id/627ecb38e4b096a69ddb81ab/base/61efb091e4b0a2be5fde0dd0%3Barchive%3B/tag/jspeed321___at___gmail___dot___com/crypt_tag/kf9ptg1jyB9ZtN_Z5Gy7DAje-NItsrQ_G0paqAtRVyJid7v3jY3aYui9_XhXB10aFakd6LUj9hmQtSTiXVtbCT/client_id/61fa64e6e4b0aba28037baef/user_id/61efb091e4b0a2be5fde0dd0/expiry_time/1726272000/rip/1609304554/file_id/627ecb38e4b096a69ddb81ae/host/cdn-eu-west-1.xvid.com/sign/e8aaa81716a8389d1bcbb223553860794d1625c8/map/true/url/627ecb38e4b096a69ddb81ab/output_0/_qIeHereo-cL4aVRMEE7mbB9tLX0SfModH_video.m3u8
https://autograph.xvid.com/api/v1/segments/job_id/627ecb38e4b096a69ddb81ab/base/61efb091e4b0a2be5fde0dd0%3Barchive%3B/tag/jspeed321___at___gmail___dot___com/crypt_tag/kf9ptg1jyB9ZtN_Z5Gy7DAje-NItsrQ_G0paqAtRVyJid7v3jY3aYui9_XhXB10aFakd6LUj9hmQtSTiXVtbCT/client_id/61fa64e6e4b0aba28037baef/user_id/61efb091e4b0a2be5fde0dd0/expiry_time/1726272000/rip/1609304554/file_id/627ecb38e4b096a69ddb81ae/host/cdn-eu-west-1.xvid.com/sign/e8aaa81716a8389d1bcbb223553860794d1625c8/map/true/url/627ecb38e4b096a69ddb81ab/output_0/_qIeHereo-cL4aVRMEE7mbB9tLX0SfModH_audio.m3u8
Mp4 b:Hls p:hls 1484.8400000000013s
AAC at unknown
H264 at 4774000, fps: ?, q:720, {"width":1280,"height":720}


</body>
</html>

**Reply by Eriaku69** [reply]
Hi, 

I have an update. If I refresh the page could happen that i see tre different version to be downloaded, one for each quality, and I am able to download them but the video downloaded is always without audio ç_ç

**Reply by GrampaWildWilly** [reply]
I do not work for VDH.  I'm just another user.  Same as Matt.

I'm running Windows 11 Pro 64-bit version 23H2 build 22631.4169, Firefox 130.0 64-bit, licensed VDH 9.1.0.41 beta, CoApp 2.0.20.

Your information contains a URL for a page on the web site alettaoceanlive.com.  This is a membership site (no news to you) so I couldn't look at the page directly myself.  However, I managed to fish 2 URLs out of the mess of details that you posted:

![#01](https://github.com/user-attachments/assets/158b2be3-6710-4016-a074-2c6bba7fffa4)

I don't know why that is such a mess.  When I copy details, I do it like this:

- Open the details.
- Ctrl+a to highlight the entire page, including parts that may be out of view.
- Ctrl+Insert to copy the details to the system clipboard.
- Switch windows to a GitHub post text box.
- Shift+Insert to paste the details in here.

When I do exactly that -- it has to be exactly that with no other steps in between -- what gets posted here is nicely formatted & shows URLs that you can just click.  Well, I dealt with it.  Using the first URL highlighted in that image, I did exactly this:

- Highlighted the URL & did Ctrl+Insert to copy the URL to the clipboard.
- Opened a new Firefox window.
- Clicked in the URL bar.
- Shift+Insert to paste the URL into the URL bar.
- Enter.
- A Firefox dialog window popped up asking what to do with the URL.  The options were to open it with an application I needed to choose & save the file.
- I did neither.  I just hit Escape to dismiss the dialog.
- VDH recognized something.
- I downloaded it.

I did this twice, once for each URL in the outlines in the image above.  I emphasize, I did these downloads with VDH.  These downloads gave me these results:

![#02](https://github.com/user-attachments/assets/81f40595-d1ab-4bff-b180-11b5815c4550)
![#03](https://github.com/user-attachments/assets/de623145-b4a5-4abc-9660-c43d0018278e)

I then played the 2 files in VLC using the synchronous playback feature of VLC (Ctrl+Shift+o).  It played just fine, both video & audio.

Since we can't visit that page, you need to be our eyes here.  We need screenshots of what VDH gives you when you visit that page.  Be sure you don't show any NSFW bits.  My expectation is that there will be multiple items on the VDH menu.  You would then need to try to download all of them.  I would hope there's only 2.  I expect one of them is the video without audio & the other is the audio without video that I just downloaded using the details information you provided.

You'll note that I am running the VDH beta & a newer CoApp.  I don't think either of those things is significant.  I think the latest general availability version of VDH & CoApp 2.0.19, both of which you are running, would work the same.

The question remains: Why doesn't VDH just recognize the integrated video instead of the tracks separately?  That is not something we've been seeing with VDH 9.  Not VDH 8, either.  That is a question for @paulrouget Paul.  Like I said, we're just users.  Paul is the developer.

**Reply by GrampaWildWilly** [reply]
Just as an aside, there is another manifest URL in front of the first one I used.  When I tried the same technique with that one, Firefox gave me an error message saying I was not permitted to view that content.  I must say I'm a bit surprised that you were not able to get this content as a single download.  That URL that failed for me ought to have been a URL that you were permitted to access since you are, evidently, a member of that web site.  That should have been an HLS master manifest that VDH should have been able to use to download the video.  I'm quite puzzled by all of this.

**Comment by Mattmjs** [comment]
I see three sets of links for each video quality, the first **api.xvid.com**. This first must be what VDH is trying to use.
It doesn't work when I try to use it, either we're not permitted to access it or it expired. There is an expiry in this url.
Or maybe there is some protection, using it directly in ffmpeg may provide more information but only @Eriaku69 can do this.
The other next two urls **autograph.xvid.com**  work.

Also the details information that is posted is easier to use if it is pasted in notepad first then copied and pasted here.
Might have to suggest this from now on.

**Reply by Eriaku69** [reply]
Hello, 

First of all, thank you a lot guys. I have the membership, so yes for me everything is visible. One thing that I forgot to say is that at beginning it wors perfectly, the download. After a few hours the issues are started. Let me show exactly what is happening. I hope a video on the site and VDH shows me just one file to be downloaded

![1](https://github.com/user-attachments/assets/00e93c35-a053-4571-8a3a-f94406605969)

I tried to download that, I get the failure message
![immagine](https://github.com/user-attachments/assets/3997625f-d33f-4d53-87e2-a5a946267b54)

Then if i refresh the page, VDH shows me 3 different files marked as audio & video 
![immagine](https://github.com/user-attachments/assets/c38c38cd-a340-4370-a84d-f1ac6ae1f86c)

but even if I download them at the same time, I get 2 videos, same content but different quality, without audio. The third file is always only audio, which is actually the entire audio of the video.

The real mistery for me is why this is suddenly starting without any apparently reason. I tried to download a video from YT, and it works as usual. 

Thanks a lot in advance.

**Reply by GrampaWildWilly** [reply]
Here's part of some details copied the way I described above:

<html><body>

    
Tab | 122 (incognito: false)
-- | --
Yuja Wang plays the Flight of the Bumble-Bee (Vol du Bourdon)
https://www.youtube.com/watch?v=8alxBofd_eQ with 0 headers
https://i.ytimg.com/vi/8alxBofd_eQ/hqdefault.jpg
false
https://manifest.googlevideo.com/api/manifest/hls_variant/expire/1726156826/ei/ubviZu6HPJ7uzN0PuIau6A8/ip/2603%3A6011%3Ac300%3Ae8c0%3Aa9e6%3Ac997%3A4e4%3A4585/id/f1a9710687ddfde4/source/youtube/requiressl/yes/xpc/EgVo2aDSNQ%3D%3D/playback_host/rr5---sn-vgqsrnz6.googlevideo.com/mh/13/mm/31%2C29/mn/sn-vgqsrnz6%2Csn-vgqsknsk/ms/au%2Crdu/mv/m/mvi/5/pl/32/hfr/1/demuxed/1/tts_caps/1/maudio/1/initcwndbps/997500/vprv/1/go/1/rqh/5/mt/1726135048/fvip/4/nvgoi/1/short_key/1/ncsapi/1/keepalive/yes/dover/13/itag/0/playlist_type/DVR/sparams/expire%2Cei%2Cip%2Cid%2Csource%2Crequiressl%2Cxpc%2Chfr%2Cdemuxed%2Ctts_caps%2Cmaudio%2Cvprv%2Cgo%2Crqh%2Citag%2Cplaylist_type/sig/AJfQdSswRQIgXor6qOMiIONwp9_fX6uZcsMCgviEymdpHTt_WQe2npQCIQD5sEavM4F8S7O3a65h1Tw_2Rl6jFGBYI68TBAbtxOtfw%3D%3D/lsparams/playback_host%2Cmh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps/lsig/ABPmVW0wRQIhAIlCqpGUn5AEppk1V1VqItO0i0JbZj7mRL4yaVD2DI_3AiAEtDS05bKMhyJP71wodv8Mio4gYIK7P2K2vPbhUqcI_Q%3D%3D/file/index.m3u8
https://manifest.googlevideo.com/api/manifest/hls_playlist/expire/1726156826/ei/ubviZu6HPJ7uzN0PuIau6A8/ip/2603:6011:c300:e8c0:a9e6:c997:4e4:4585/id/f1a9710687ddfde4/itag/231/source/youtube/requiressl/yes/ratebypass/yes/pfa/1/sgovp/clen%3D4740069%3Bdur%3D102.400%3Bgir%3Dyes%3Bitag%3D135%3Blmt%3D1703453835482444/rqh/1/hls_chunk_host/rr5---sn-vgqsrnz6.googlevideo.com/xpc/EgVo2aDSNQ%3D%3D/mh/13/mm/31,29/mn/sn-vgqsrnz6,sn-vgqsknsk/ms/au,rdu/mv/m/mvi/5/pl/32/initcwndbps/997500/vprv/1/playlist_type/DVR/dover/13/txp/5432434/mt/1726135048/fvip/4/short_key/1/keepalive/yes/sparams/expire,ei,ip,id,itag,source,requiressl,ratebypass,pfa,sgovp,rqh,xpc,vprv,playlist_type/sig/AJfQdSswRQIhAPA4H-OguQG9SN2oqIaQlPBELEVrbej5qi-C6WV8g-McAiATF1I_nYg0VoZqF9Erc1UKIVkwq-pCcLG0BM9t_3ub7A%3D%3D/lsparams/hls_chunk_host,mh,mm,mn,ms,mv,mvi,pl,initcwndbps/lsig/ABPmVW0wRgIhAMxotgFPovqBiSD1Cni-jVIMm1-TY1jgWINW7d4dCcwUAiEAg_JjVEohBttGbVQhY6lf4qA_iSYnOxRmBT-3MyTGxdc%3D/playlist/index.m3u8
https://manifest.googlevideo.com/api/manifest/hls_playlist/expire/1726156826/ei/ubviZu6HPJ7uzN0PuIau6A8/ip/2603:6011:c300:e8c0:a9e6:c997:4e4:4585/id/f1a9710687ddfde4/itag/234/source/youtube/requiressl/yes/ratebypass/yes/pfa/1/goi/133/sgoap/clen%3D1659857%3Bdur%3D102.516%3Bgir%3Dyes%3Bitag%3D140%3Blmt%3D1703453339778762/rqh/1/hls_chunk_host/rr5---sn-vgqsrnz6.googlevideo.com/xpc/EgVo2aDSNQ%3D%3D/mh/13/mm/31,29/mn/sn-vgqsrnz6,sn-vgqsknsk/ms/au,rdu/mv/m/mvi/5/pl/32/initcwndbps/997500/vprv/1/playlist_type/DVR/dover/13/txp/5432434/mt/1726135048/fvip/4/short_key/1/keepalive/yes/sparams/expire,ei,ip,id,itag,source,requiressl,ratebypass,pfa,goi,sgoap,rqh,xpc,vprv,playlist_type/sig/AJfQdSswRQIgMq9kcjcUkfsIr1OJjLI3YvmNl69-DuFIqJsAfKa0CeECIQC8694Vt0DzBWmAbGfzPcPIIp4efY-6ZRotUmXBlhLvdQ%3D%3D/lsparams/hls_chunk_host,mh,mm,mn,ms,mv,mvi,pl,initcwndbps/lsig/ABPmVW0wRQIgdU2FjK3Yw3XBUVt6hgh310PC1y2KoNtbdkTvQWadUyYCIQD1R9yK4nE0_BPBh34xJM8wBOvn1bl6XMzSlsKBuHCZlg%3D%3D/playlist/index.m3u8
Mp4 b:Hls p:hls 102.4s
AAC at unknown
H264 at 681054, fps: 25, q:360, {"width":854,"height":478}


</body>
</html>

And here's what happens when I pass that through Notepad++ first:

Tab	122 (incognito: false)
Title	Yuja Wang plays the Flight of the Bumble-Bee (Vol du Bourdon)
Page URL	https://www.youtube.com/watch?v=8alxBofd_eQ with 0 headers
Thumbnail	https://i.ytimg.com/vi/8alxBofd_eQ/hqdefault.jpg
Low Quality	false
#0 main url	https://manifest.googlevideo.com/api/manifest/hls_variant/expire/1726156826/ei/ubviZu6HPJ7uzN0PuIau6A8/ip/2603%3A6011%3Ac300%3Ae8c0%3Aa9e6%3Ac997%3A4e4%3A4585/id/f1a9710687ddfde4/source/youtube/requiressl/yes/xpc/EgVo2aDSNQ%3D%3D/playback_host/rr5---sn-vgqsrnz6.googlevideo.com/mh/13/mm/31%2C29/mn/sn-vgqsrnz6%2Csn-vgqsknsk/ms/au%2Crdu/mv/m/mvi/5/pl/32/hfr/1/demuxed/1/tts_caps/1/maudio/1/initcwndbps/997500/vprv/1/go/1/rqh/5/mt/1726135048/fvip/4/nvgoi/1/short_key/1/ncsapi/1/keepalive/yes/dover/13/itag/0/playlist_type/DVR/sparams/expire%2Cei%2Cip%2Cid%2Csource%2Crequiressl%2Cxpc%2Chfr%2Cdemuxed%2Ctts_caps%2Cmaudio%2Cvprv%2Cgo%2Crqh%2Citag%2Cplaylist_type/sig/AJfQdSswRQIgXor6qOMiIONwp9_fX6uZcsMCgviEymdpHTt_WQe2npQCIQD5sEavM4F8S7O3a65h1Tw_2Rl6jFGBYI68TBAbtxOtfw%3D%3D/lsparams/playback_host%2Cmh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps/lsig/ABPmVW0wRQIhAIlCqpGUn5AEppk1V1VqItO0i0JbZj7mRL4yaVD2DI_3AiAEtDS05bKMhyJP71wodv8Mio4gYIK7P2K2vPbhUqcI_Q%3D%3D/file/index.m3u8
#0 video component	https://manifest.googlevideo.com/api/manifest/hls_playlist/expire/1726156826/ei/ubviZu6HPJ7uzN0PuIau6A8/ip/2603:6011:c300:e8c0:a9e6:c997:4e4:4585/id/f1a9710687ddfde4/itag/231/source/youtube/requiressl/yes/ratebypass/yes/pfa/1/sgovp/clen%3D4740069%3Bdur%3D102.400%3Bgir%3Dyes%3Bitag%3D135%3Blmt%3D1703453835482444/rqh/1/hls_chunk_host/rr5---sn-vgqsrnz6.googlevideo.com/xpc/EgVo2aDSNQ%3D%3D/mh/13/mm/31,29/mn/sn-vgqsrnz6,sn-vgqsknsk/ms/au,rdu/mv/m/mvi/5/pl/32/initcwndbps/997500/vprv/1/playlist_type/DVR/dover/13/txp/5432434/mt/1726135048/fvip/4/short_key/1/keepalive/yes/sparams/expire,ei,ip,id,itag,source,requiressl,ratebypass,pfa,sgovp,rqh,xpc,vprv,playlist_type/sig/AJfQdSswRQIhAPA4H-OguQG9SN2oqIaQlPBELEVrbej5qi-C6WV8g-McAiATF1I_nYg0VoZqF9Erc1UKIVkwq-pCcLG0BM9t_3ub7A%3D%3D/lsparams/hls_chunk_host,mh,mm,mn,ms,mv,mvi,pl,initcwndbps/lsig/ABPmVW0wRgIhAMxotgFPovqBiSD1Cni-jVIMm1-TY1jgWINW7d4dCcwUAiEAg_JjVEohBttGbVQhY6lf4qA_iSYnOxRmBT-3MyTGxdc%3D/playlist/index.m3u8
#0 audio component	https://manifest.googlevideo.com/api/manifest/hls_playlist/expire/1726156826/ei/ubviZu6HPJ7uzN0PuIau6A8/ip/2603:6011:c300:e8c0:a9e6:c997:4e4:4585/id/f1a9710687ddfde4/itag/234/source/youtube/requiressl/yes/ratebypass/yes/pfa/1/goi/133/sgoap/clen%3D1659857%3Bdur%3D102.516%3Bgir%3Dyes%3Bitag%3D140%3Blmt%3D1703453339778762/rqh/1/hls_chunk_host/rr5---sn-vgqsrnz6.googlevideo.com/xpc/EgVo2aDSNQ%3D%3D/mh/13/mm/31,29/mn/sn-vgqsrnz6,sn-vgqsknsk/ms/au,rdu/mv/m/mvi/5/pl/32/initcwndbps/997500/vprv/1/playlist_type/DVR/dover/13/txp/5432434/mt/1726135048/fvip/4/short_key/1/keepalive/yes/sparams/expire,ei,ip,id,itag,source,requiressl,ratebypass,pfa,goi,sgoap,rqh,xpc,vprv,playlist_type/sig/AJfQdSswRQIgMq9kcjcUkfsIr1OJjLI3YvmNl69-DuFIqJsAfKa0CeECIQC8694Vt0DzBWmAbGfzPcPIIp4efY-6ZRotUmXBlhLvdQ%3D%3D/lsparams/hls_chunk_host,mh,mm,mn,ms,mv,mvi,pl,initcwndbps/lsig/ABPmVW0wRQIgdU2FjK3Yw3XBUVt6hgh310PC1y2KoNtbdkTvQWadUyYCIQD1R9yK4nE0_BPBh34xJM8wBOvn1bl6XMzSlsKBuHCZlg%3D%3D/playlist/index.m3u8
#0 core media	Mp4 b:Hls p:hls 102.4s
#0 audio	AAC at unknown
#0 video	H264 at 681054, fps: 25, q:360, {"width":854,"height":478}

Not much difference.  I don't think passing it through a text editor is necessary.  But I don't know why some of these hit details show up in here like a dog's breakfast.  There must be something interfering but I can't say what.

**Reply by GrampaWildWilly** [reply]
@Eriaku69 You have at least one file that is video without audio, & at least one other file that is audio without video.  Now I want you to go into VLC & do Ctrl+Shift+o.  Then on that dialog, do this:

![#01](https://github.com/user-attachments/assets/5f5a72bb-ea59-4323-9c09-2a7674f2dda9)
![#02](https://github.com/user-attachments/assets/b8c209a0-1d14-41a5-a2e2-d204291ad2e2)
![#03](https://github.com/user-attachments/assets/360d233b-ee7f-4ebf-9100-24ada8cd9a60)

Tell us what happens.

**Comment by Eriaku69** [comment]
Hello, 

Sorry this delay! I have done what you have explained willy, and the video in VLC is played correctly with the audio!
There is a way to save this, or I have to do that each time?

**Reply by GrampaWildWilly** [reply]
Apparently, VDH has a problem recognizing the content on this site.  I don't know what can be done about that.  We'll just have to wait for a bug fix in VDH to correct that.  In the meantime, it seems you will have to obtain the video & audio separately.

You could continue to play the video & audio tracks separately using this feature of VLC.  It is called synchronous playback.  You don't really need to merge the video with the audio.  You might want to but it's not necessary.

In older versions of VDH, there used to be a tool for merging video & audio tracks into a single file.  This functionality has not been ported forward to VDH 9.  But it is ridiculously simple to merge a video track with its audio track.  You have to execute ffmpeg at a command prompt.  That is easy to do because ffmpeg is in the same directory with the CoApp.  It is distributed as part of the CoApp, & the CoApp installer places ffmpeg in whatever directory you chose to install the CoApp.  You can find it easily enough by executing `dir c:\ffmpeg.exe /s` at a command prompt.  Once you know where ffmpeg is on your system, just execute this:

"path to ffmepg\ffmpeg" -i "file name of video track" -i "file name of audio track" -codec: ccopy "file name of merged video"

You must supply the file names of 4 files there.  Be sure you surround them with " " to make sure there's no surprises.  The other parts of the command must be coded exactly as I show them in that example.  My notation `"path to ffmpeg\ffmpeg"` might be `"C:\Program Files\ffmpeg"` or something like that.  Use whatever path you discover for where ffmpeg is located.  The other 3 files can be anywhere on your system.  They could potentially be in 3 different directories, although I suspect you'll choose to keep them all in the same directory.  Just be sure you code complete file specifications in the form `"x:\directory\directory\directory\file name"`.

**Reply by Eriaku69** [reply]
> Apparently, VDH has a problem recognizing the content on this site. I don't know what can be done about that. We'll just have to wait for a bug fix in VDH to correct that. In the meantime, it seems you will have to obtain the video & audio separately.
> 
> You could continue to play the video & audio tracks separately using this feature of VLC. It is called synchronous playback. You don't really need to merge the video with the audio. You might want to but it's not necessary.
> 
> In older versions of VDH, there used to be a tool for merging video & audio tracks into a single file. This functionality has not been ported forward to VDH 9. But it is ridiculously simple to merge a video track with its audio track. You have to execute ffmpeg at a command prompt. That is easy to do because ffmpeg is in the same directory with the CoApp. It is distributed as part of the CoApp, & the CoApp installer places ffmpeg in whatever directory you chose to install the CoApp. You can find it easily enough by executing `dir c:\ffmpeg.exe /s` at a command prompt. Once you know where ffmpeg is on your system, just execute this:
> 
> "path to ffmepg\ffmpeg" -i "file name of video track" -i "file name of audio track" -codec: ccopy "file name of merged video"
> 
> You must supply the file names of 4 files there. Be sure you surround them with " " to make sure there's no surprises. The other parts of the command must be coded exactly as I show them in that example. My notation `"path to ffmpeg\ffmpeg"` might be `"C:\Program Files\ffmpeg"` or something like that. Use whatever path you discover for where ffmpeg is located. The other 3 files can be anywhere on your system. They could potentially be in 3 different directories, although I suspect you'll choose to keep them all in the same directory. Just be sure you code complete file specifications in the form `"x:\directory\directory\directory\file name"`.

Hi! 

I have done and it works! I want just to say thank you both. I am feel really lucky to have found people who helps just for help. Again, thanks a lot!
